custom_directives.py

```diff
@@ -92,7 +92,7 @@ def run(self):
                 intro, _ = sphinx_gallery.gen_rst.extract_intro_and_title(abs_fname, blocks[0][1])
 
             thumbnail_rst = sphinx_gallery.backreferences._thumbnail_div(
-                dirname, basename, intro)
+                dirname, dirname, basename, intro)
```

Comment:
is this change necessary?

Reply:
this was required to generate the html files locally. I was testing to see if it was the reason why the CI was failing.
I reverted the change now.